web-api/nishiki-web-api.yaml

```
@@ -404,6 +404,8 @@ paths:
       responses:
         "204":
           description: deletion complete
+        "403":
+          description: A user tries to delete a container that is not owned by a requesting user.
```

Could you change the objective noun of the deletion?
```suggestion
          description: A user tries to delete food belonging to a container that is not owned by a requesting user.
```